### ISSUE TITLE
Incomplete installation instructions

### ISSUE THREAD

**Issue by fkromer** [jaydenwindle/graphene-subscriptions#2]
In the [REAME installation instructions](https://github.com/jaydenwindle/graphene-subscriptions#installation), step 6. w.r.t. `from your_app.graphql.subscriptions import YourSubscription` something is missing.

**Comment by jaydenwindle** [comment]
@fkromer You're right, the instructions refer to a subscription called `YourSubscription` imported from `your_app.graphql.subscriptions` that isn't defined in the instructions. I can definitely see how that would be confusing.

I like your alternative, would you like to open a PR to add it to the README?

**Comment by fkromer** [comment]
@jaydenwindle I'll try to find some time to create a PR.

Thanks for this package by the way. The best plug-and-play feel to add subscription support to django so far!

**Comment by jaydenwindle** [comment]
Awesome, thanks!

You're very welcome, glad you like it! Let me know if you run into any issues 😄
